<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEditCameraROI</class>
 <widget class="QWidget" name="WidgetEditCameraROI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frameDashBoardROIHoseDetectionXPosition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>82</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
  
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
 border: px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_17">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardROIHoseDetectionXPosition">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ROI X Position</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxROIHoseDetectionXPosition">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> pix</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frameDashBoardROIHoseDetectionYPosition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>82</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
   
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_18">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardROIHoseDetectionYPosition">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ROI Y Position</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxROIHoseDetectionYPosition">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> pix</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frameDashBoardROIHoseDetectionWidth">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>82</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
    
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_19">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardROIHoseDetectionWidth">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ROI Width</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxROIHoseDetectionWidth">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> pix</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFrame" name="frameDashBoardROIHoseDetectionHeight">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>82</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
     background:rgb(249,247,248);
    
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_20">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelDashBoardROIROIHoseDetectionHeight">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ROI Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxROIHoseDetectionHeight">
          <property name="font">
           <font>
            <family>Siemens Sans</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> pix</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>5000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>49</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
